pkgs/desktops/gnome-3/core/gnome-contacts/default.nix

```diff
@@ -7,11 +7,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
aargh, can't mark the part of the diff I mean, but the comment about telepathy integration not building isn't exactly true since this release. So it just needs to be updated that we "don't want it".